postprocessing/visualization/receiver/src/Filters.py

```diff
@@ -180,3 +180,30 @@ def apply(self, wf):
       wf.waveforms['radial'] = math.cos(phi) * u + math.sin(phi) * v
       wf.waveforms['transverse'] = -math.sin(phi) * u + math.cos(phi) * v
       wf.waveforms['vertical'] = w
+      for new_comp in ['radial', 'transverse', 'vertical']:
+        wf.show[new_comp] = True
+
+class Pick(Filter):
+  def __init__(self, parent = None):
+    super(Pick, self).__init__('Pick components', parent)
+    self.cb_widget_list = []
+
+  def initialize_checkboxes(self, wf):
```

Comment:
Maybe `create_checkboxes`. IMHO initialize would be to set default values to something, which already exists.